index.js

```diff
@@ -3104,7 +3104,7 @@
   def('trim',
       {},
       [$.String, $.String],
-      R.trim);
+      function(s) { return s.trim(); });
```

Comment:
I guess that's David's call. I look at the Ramda implementation and it seems insane to go to those lengths to support a browser like IE8.